h2o-core/src/main/java/water/H2O.java

```diff
@@ -292,6 +292,9 @@ public static void printHelp() {
     /** -context_path=jetty_context_path; the context path for jetty */
     public String context_path = "";
 
+    /** -external_xgboost_cluster localhost:12345 TESTING ONLY */
+    public String externalXGBoostClusterAddress;
```

Comment:
I would rather use a system property instead of using a parameter just for development

Reply:
done